end_to_end/llama_load_and_test.sh

```diff
@@ -13,7 +13,7 @@ export M_ASYNC_CHECKPOINTING=false
 #python3 MaxText/llama_or_mistral_ckpt.py --base-model-path /tmp/meta-ckpt --model-size llama2-7b --maxtext-model-path gs://maxtext-llama/test/${idx}/decode-ckpt-maxtext/
 
 # Load after directly from parameter checkpoint
-python3 MaxText/decode.py MaxText/configs/base.yml load_parameters_path=${base_ckpt_path} run_name=runner_direct_${idx} per_device_batch_size=1 model_name='llama2-7b' assets_path=gs://maxtext-llama/llama2-7b ici_tensor_parallelism=4 max_prefill_predict_length=4  max_target_length=16 prompt="I love to" autoregressive_decode_assert="read. I love to write. I love to share." attention=dot_product
```

Comment:
please don't change this -- actually the fix is to remove attention=dot_product from the llama model. (Decode right now wants attention=dot_product but for some reason it is being set at the llama2 level)

Reply:
Sure, I will do the change accordingly.